recipes/latlon23/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "LatLon23" %}
+{% set version = "1.0.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e0ea06eda0ac1dab658fec89ee158c5221948cf147e38724fe50dfbd8a1ff340
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - pyproj
+    - python
+  run:
+    - pyproj
+    - python
+
+test:
+  imports:
+    - LatLon23
+
+about:
+  home: https://github.com/hickeroar/LatLon23
+  license: GNU General Public v3 (GPLv3)
```

From https://opensource.org/licenses/alphabetical
```suggestion
  license: GPL-3.0
```